app/components/phone_input_component.rb

```diff
@@ -66,9 +69,21 @@ def international_phone_code_label(code_data)
   end
 
   def international_phone_codes_data(code_data)
+    supports_sms = code_data['supports_sms']
+    supports_sms_unconfirmed = code_data.fetch(
+      'supports_sms_unconfirmed',
+      supports_sms,
+    )
+
+    supports_voice = code_data['supports_voice']
+    supports_voice_unconfirmed = code_data.fetch(
+      'supports_voice_unconfirmed',
+      supports_voice,
+    )
```

Comment:
Styling nit (maybe personal preference, your call): We could drop some newlines

```suggestion
    supports_sms = code_data['supports_sms']
    supports_sms_unconfirmed = code_data.fetch('supports_sms_unconfirmed', supports_sms)

    supports_voice = code_data['supports_voice']
    supports_voice_unconfirmed = code_data.fetch('supports_voice_unconfirmed', supports_voice)
```